bin/ch-fromhost

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Note: The ${x:+y} construction yields $y if $x is set and not null, or $x
+# otherwise; here, to append a delimiter on second and subsequent iterations.
+# See: https://chris-lamb.co.uk/posts/joining-strings-in-posix-shell
+
+LIBEXEC="$(cd "$(dirname "$0")" && pwd)"
+. ${LIBEXEC}/base.sh
+
+usage () {
+    cat 1>&2 <<EOF
+Usage: FIXME
+EOF
+    exit ${1:-1}
+}
+
+DEST_DEFAULT=
+IMAGE=
+INFER=yes
+NEWLINE='
```

Comment:
The more correct way to say this in `bash` is `$'\n'`.

Reply:
The script should be targeting POSIX sh, not Bash. `/bin/bash` and all the bashisms are a mistake. (Currently we claim that using Charliecloud requires only sh, but the test suite needs Bash.)

Reply:
POSIX `sh` does not appear to support `$'...'` constructs, so I guess it should stay as-is.  But I don't believe `[[ ... ]]` is POSIX either, so those would need to be changed.  I think `${DEST:0:1}` is also `bash`-specific.